wurst/network/GamecacheKeys.wurst

```diff
@@ -0,0 +1,33 @@
+package GamecacheKeys
+
+import Execute
+import ErrorHandling
+import Network
+
+// characters that can be used as GC keys
+// their amount also determines the size of a single 'round'
+constant string GC_KEYS = " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`{|}~"
+
+public class GamecacheKeys
+	static string array keys
+	static var count = 0
+
+	static function getKey(int i) returns string
```

Comment:
if the class is already name **Keys this should just be "get"

Reply:
Done